app/controllers/verify/review_controller.rb

```diff
@@ -23,6 +23,13 @@ def confirm_current_password
     def new
       idv_session.params.symbolize_keys!
       analytics.track_event(Analytics::IDV_REVIEW_VISIT)
+
+      phone_of_record_msg = ActionController::Base.helpers.content_tag(
+        :strong, t('idv.messages.phone.phone_of_record')
+      )
+
+      flash[:success] = t('idv.messages.review.info_verified_html',
```

Comment:
This should be `flash.now[:success]` so that the flash message does not persist after entering your password.